implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -152,7 +168,9 @@ impl CreateCommand {
 
 async fn setup(ctx: Context, (c, cfg): (CreateCommand, OckamConfig)) -> anyhow::Result<()> {
     let tcp = TcpTransport::create(&ctx).await?;
-    let bind = c.api_address;
+    // We can unwrap `c.api_address` as by this point,
+    // an available port was found, and used to create an api_address in `run`
+    let bind = c.api_address.unwrap();
```

Comment:
This `unwrap` is a result of changing `api_address` to an Option, which was done because we no longer have a static default value.  While this is sound, I could also leave the default value, check if it is the value of `api_address` and update the `api_address` accordingly with a new port to be used.  That feels worse to me, but I open to changing to that strategy if we would prefer to avoid this unwrap.

Reply:
What I like about making the default value `127.0.0.1:0` and that showing up in the cli help for `ockam node create` is that it communicates to the used what will happen.

Reply:
That makes sense to me, will make that change

Reply:
Do you want to do that in this PR or in a different followup PR? Since this is about to pass checks, we can merge it if you want.

Reply:
Whoops, didn't see your comment, included the change in this PR.  (I'm used to my full time job where a change like this would be included in the original PR and not in a follow-up)

Reply:
either way works. thank you